src/python/grpcio_tests/tests_aio/unit/init_test.py

```diff
@@ -14,38 +14,26 @@
 import logging
 import unittest
 
-import grpc
-
-from grpc.experimental import aio
-from tests_aio.unit._test_server import start_test_server
 from tests_aio.unit._test_base import AioTestBase
 
-from tests.unit import resources
-
-_PRIVATE_KEY = resources.private_key()
-_CERTIFICATE_CHAIN = resources.certificate_chain()
-_TEST_ROOT_CERTIFICATES = resources.test_root_certificates()
 
+class TestInit(AioTestBase):
 
-class TestChannel(AioTestBase):
+    async def test_grpc(self):
+        import grpc  # pylint: disable=wrong-import-position
+        channel = grpc.aio.insecure_channel('dummy')
+        self.assertIsInstance(channel, grpc.aio.Channel)
 
-    async def test_insecure_channel(self):
-        server_target, _ = await start_test_server()  # pylint: disable=unused-variable
+    async def test_grpc_dot_aio(self):
+        import grpc.aio  # pylint: disable=wrong-import-position
+        channel = grpc.aio.insecure_channel('dummy')
+        self.assertIsInstance(channel, grpc.aio.Channel)
 
-        channel = aio.insecure_channel(server_target)
+    async def test_aio_from_grpc(self):
+        from grpc import aio  # pylint: disable=wrong-import-position
+        channel = aio.insecure_channel('dummy')
         self.assertIsInstance(channel, aio.Channel)
 
-    async def test_secure_channel(self):
```

Comment:
Any particular reason this test case is being removed?

Reply:
The secure channel case is well tested in [secure_call_test.py](https://github.com/grpc/grpc/blob/master/src/python/grpcio_tests/tests_aio/unit/secure_call_test.py), and the insecure channel case is tested in almost all other "unit" tests. I think this file wasn't in a good shape as you can see the conflict between the file name and the test class name. But it has a good name for actual module initialization tests.